lib/config/app_config.dart

```diff
@@ -74,6 +74,10 @@ abstract class AppConfig {
   static const int thumbnailQuality = 70;
   static const int blurHashSize = 32;
   static const int imageQuality = 50;
+  static const String iOSKeychainSharingId =
+      'KUT463DS29.com.linagora.ios.twake.shared';
```

Comment:
what ?

Reply:
do we need to have `KUT463DS29` prefix

Reply:
yes 
https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps/#2974916